src/keycloak/chart/values.yaml

```diff
@@ -49,6 +49,11 @@ clusterDomain: cluster.local
 # Sets development mode for Keycloak. This disables caching, Postgres and HPAs and should only be used for testing
 devMode: true
 
+# Enable SMTP networkPolicy and config
+smtp:
+  enabled: false
+  port: 587
+
```

Comment:
Not requesting a change here, but noting for the future we can use this values block to add address/other things we may need to pass the smtp server to the realm once that is exposed on that side - cc @UnicornChance